lib/optimizer/src/Transformer/ServerSideRendering.php

```diff
@@ -953,7 +953,10 @@ private function extractSizesAttributeCss(Document $document, DOMElement $elemen
      */
     private function extractHeightsAttributeCss(Document $document, DOMElement $element, DOMAttr $attribute)
     {
-        $this->customSizerStyles[$document->getElementId($element)] = '';
+        // TODO: I'm not sure why I initially added this here, it looks very intentional.
+        // However, it doesn't match what the NodeJS version does, which is to add padding-top
+        // to the inline style of the element.
+        // $this->customSizerStyles[$document->getElementId($element)] = '';
```

Comment:
This change is what's needed to get in sync with the NodeJS version. However, the entire code construct for `$this->customSizerStyles` seems to have only been included for skipping the styles here, and I don't know why I added this in the first place, so I'm unsure about this "fix" and wonder whether the NodeJS version is correct in the first place.
I'll revisit this as soon as we have Percy visual regression tests, to diligently compare non-optimized and optimized versions of everything.